test/asmt-double-entry.js

```diff
@@ -34,8 +74,8 @@ describe('navigate to asmt and fill out asmts', function() {
             instrumentId: 26363, // Calculation Test
             segmentInterval: 'v3', // can vary this: visit1, v1, v2, v3, v4
             sourceType: '5',  // can vary this: 3, 4, 5, 6, 7, 8, 9
-            assessmentDate: '12/24/1900', // don't change this Date, or need to make sure it's unique in resume queue
-            assessmentStartTime: '10:22',
+            assessmentDate: getRandomDate(),
```

Comment:
We are saving the random date/time to an object property for reuse later in the test. This was working for me while running tests. Let me know if it does not work for you when you run the tests.